apps/io-func-wallet-solution-backend/src/app/main.ts

```diff
@@ -74,3 +78,12 @@ app.http("getNonce", {
   route: "nonce",
   handler: GetNonceFunction({ nonceRepository }),
 });
+
+app.http("getUserIdByFiscalCode", {
+  methods: ["POST"],
+  authLevel: "anonymous",
```

Comment:
why anonymous? shouldn't it use a function key?

Reply:
yes in another task I will convert all `authLevel` to `function`